### ISSUE TITLE
Incorrect Documentation for MPI example with 3.6

### ISSUE THREAD

**Issue by ross-chaudhry** [apptainer/singularity#5839]
### Version of Singularity:
```bash
$ singularity --version
singularity version 3.6.4
```

### Expected behavior
The instructions for setting up an OpenMPI environment using version 3.6 [https://sylabs.io/guides/3.6/user-guide/mpi.html](here) include an `%environment` section with variables like `SINGULARITYENV_APPEND_PATH`. A minimal example is:
```bash
PROMPT$ cat Test_Environment.def 
Bootstrap: library
From: ubuntu:18.04

%environment
   # -- Set OpenMPI PATHs according to 3.6 user guide https://sylabs.io/guides/3.6/user-guide/mpi.html
   export OMPI_DIR=/opt/ompi
   export SINGULARITY_OMPI_DIR=$OMPI_DIR
   export SINGULARITYENV_APPEND_PATH=$OMPI_DIR/bin
   export SINGULARITYENV_APPEND_LD_LIBRARY_PATH=$OMPI_DIR/lib

   # -- Follow 3.7 guide instead https://sylabs.io/guides/3.7/user-guide/mpi.html
   export PATH="$PATH:/actually/appended/path"
   export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/actually/appended/ld/path"
```
I expect these instructions to be valid: `$PATH` should be appended with `/opt/ompi` and `$LD_LIBRARY_PATH` should be appended with `/opt/ompi/lib` when executing in the singularity environment.

### Actual behavior
The variables `SINGULARITY_*` exist directly in execution environment, and the paths are instead appended by variables set using the guide shown for 3.7.
```bash
PROMPT$ singularity build --remote Test_Environment.sif Test_Environment.def 
INFO:    API Key Verified!
...
INFO:    Build complete: Test_Environment.sif

PROMPT$ singularity exec Test_Environment.sif env | grep PATH
LD_LIBRARY_PATH=:/actually/appended/ld/path:/.singularity.d/libs
...
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/actually/appended/path
SINGULARITYENV_APPEND_LD_LIBRARY_PATH=/opt/ompi/lib
SINGULARITYENV_APPEND_PATH=/opt/ompi/bin
...
```
This behavior seems to be incorrect based on the 3.6 MPI documentation. I guess that the 3.6 MPI documentation was not updated with environment changes that occured with 3.6 [https://sylabs.io/guides/3.6/user-guide/environment_and_metadata.html](referenced here).

### Steps to reproduce this behavior
Using singularity version 3.6, build and execute `env` on the sample definition file `Test_Environment.def` shown above.

### What OS/distro are you running and how did you install Singularity
I am using singularity in a managed HPC environment running red hat 7.4. I did not install it myself.

**Comment by dtrudg** [comment]
I'm afraid that the open source documentation is generally updated for the current version of Singularity only. There were errors found in the MPI documentation that were fixed in the 3.7 release cycle, but we do not generally port them back to prior versions of the docs.

It is part related to the 3.6 environment changes, but the MPI docs had errors for prior versions also. Apologies for the inconvenience.

**Comment by ross-chaudhry** [comment]
Thanks for the response. It too bad that this is the policy - version 3.6 is not very old, and a diligent user is likely to target the documentation of the version they're currently running. In a managed HPC environment, it's not practical for me to be always running the latest version of Singularity.

~~If I pushed a fix for this, would it get merged in?~~ Eh, I guess if somebody wants to update to get this fix they should just update to 3.7.X.

**Comment by dtrudg** [comment]
> ~If I pushed a fix for this, would it get merged in?~ Eh, I guess if somebody wants to update to get this fix they should just update to 3.7.X.

Happy to build and upload updated docs if a fix is submitted to the docs repo: https://github.com/hpcng/singularity-userdocs/

In the future we will build docs from the repo to a hosted location automatically, so community contributions to update/fix older docs will be more straightforward.

**Comment by dtrudg** [comment]
Going to close this issue here on the code repository. If someone would like to backport the docs fixes an issue / PR would be appropriate against our userdocs repo: https://github.com/hpcng/singularity-userdocs/
